__tests__/api.test.js

```diff
@@ -258,4 +258,36 @@ describe('8. PATCH /api/reviews/:review_id', () => {
             expect(res.body.msg).toBe('Bad request');
         })
     });
+});
+
+describe('GET /api/users', () => {
+    test('gets users', () => {
+        return request(app)
+        .get('/api/users')
+        .expect(200)
+        .then((users) => {
+            expect(users.body).toEqual([
+                {
+                  username: 'mallionaire',
+                  name: 'haz',
+                  avatar_url: 'https://www.healthytherapies.com/wp-content/uploads/2016/06/Lime3.jpg'
+                },
+                {
+                  username: 'philippaclaire9',
+                  name: 'philippa',
+                  avatar_url: 'https://avatars2.githubusercontent.com/u/24604688?s=460&v=4'
+                },
+                {
+                  username: 'bainesface',
+                  name: 'sarah',
+                  avatar_url: 'https://avatars2.githubusercontent.com/u/24394918?s=400&v=4'
+                },
+                {
+                  username: 'dav3rid',
+                  name: 'dave',
+                  avatar_url: 'https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png'
+                }
+              ])
+        })
+    });
```

Comment:
Is this for ticket 9?
I would use the same testing approach as all comments or all reviews - not being too rigid here